relay-server/src/actors/events.rs

```diff
@@ -519,6 +520,7 @@ impl EventProcessor {
             remove_other: Some(true),
             normalize_user_agent: Some(true),
             sent_at: envelope.sent_at(),
+            received_at: Some(relay_common::instant_to_date_time(start_time)),
```

Comment:
We convert from `Instant` to `DateTime` as close as possible to normalization to ensure we have an accurate clock.